components/keyboard_shortcuts/keyboard_shortcuts_sequence/keyboard_shortcuts_sequence.tsx

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import {FormatXMLElementFn, PrimitiveType} from 'intl-messageformat';
+import React, {memo} from 'react';
+import {useIntl} from 'react-intl';
+
+import {ShortcutKetVariant, ShortcutKey} from '../../shortcut_key';
+
+import {isMac} from 'utils/utils';
+
+import {isMessageDescriptor, KeyboardShortcutDescriptor} from './keyboard_shortcuts';
+
+import './keyboard_shortcuts_sequence.scss';
+
+type Props = {
+    shortcut: KeyboardShortcutDescriptor;
+    values?: Record<string, PrimitiveType | FormatXMLElementFn<string, string>>;
+    hideDescription?: boolean;
+    hoistDescription?: boolean;
+    isInsideTooltip?: boolean;
+};
+
+function normalizeShortcutDescriptor(shortcut: KeyboardShortcutDescriptor) {
+    if (isMessageDescriptor(shortcut)) {
+        return shortcut;
+    }
+
+    const {default: standard, mac} = shortcut;
+    return isMac() && mac ? mac : standard;
+}
```

Comment:
IMO calling this every time seems a bit much. I would prefer to have a `const` with the property string intiated outside the scope and just applied here.

Something like this:

```suggestion
const descriptorProperty = isMac() ? 'mac' : 'default';

function normalizeShortcutDescriptor(shortcut: KeyboardShortcutDescriptor) {
    if (isMessageDescriptor(shortcut)) {
        return shortcut;
    }

    return shortcut[descriptorProperty];
}
```

Reply:
@michelengelen may be something like this, as we have to check if mac isn't undefined 
```
const isOsMac = isMac();
function normalizeShortcutDescriptor(shortcut: KeyboardShortcutDescriptor) {
    if (isMessageDescriptor(shortcut)) {
        return shortcut;
    }
    const {default: standard, mac} = shortcut;
    return isOsMac && mac ? mac : standard;
}
```

Reply:
that is slightly better, but can still be improved by doing it like this:

```typescript
const descriptorProperty = isMac() ? 'mac' : 'default';

function normalizeShortcutDescriptor(shortcut: KeyboardShortcutDescriptor) {
    if (isMessageDescriptor(shortcut)) {
        return shortcut;
    }

    // return the default value when `shortcut.mac` is undefined
    return shortcut[descriptorProperty] || shortcut.default;
}
```